packages/cli-lib/src/extract.ts

```diff
@@ -217,7 +217,9 @@ export async function extract(
     }
   }
 
-  const formatter: Formatter = await resolveBuiltinFormatter(opts.format)
+  const formatter: Formatter<unknown> = await resolveBuiltinFormatter(
```

Comment:
Previously, `formatter` was instantiated
```
{
  serialize?: (msgs: Record<string, MessageDescriptor>) => string
  format: (msgs: Record<string, MessageDescriptor>) => Record<string, MessageDescriptor>
  compile: (msgs: Record<string, MessageDescriptor>) => Record<string, string>
  compareMessages?: Comparator
}
```
for every formatter, while in runtime it was different. E.g., for `smartling`:
```
{
  serialize?: (msgs: SmartlingJson) => string
  format: (msgs: Record<string, MessageDescriptor>) => SmartlingJson
  compile: (msgs: SmartlingJson) => Record<string, string>
  compareMessages?: Comparator
}
```
etc

So, having less type information is more correct in that case